src/java/org/apache/cassandra/db/virtual/SettingsTable.java

```diff
@@ -59,102 +56,83 @@ final class SettingsTable extends AbstractVirtualTable
-     * fixes this and returns the proper (what is accessable via yaml) names.
-     */
-    private static Map<String, String> getBackwardsCompatableNames()
+    @Override
+    protected void applyColumnUpdate(ColumnValues partitionKey,
+                                     ColumnValues clusteringColumns,
+                                     Optional<ColumnValue> columnValue)
     {
-        Map<String, String> names = new HashMap<>();
-        // Names that dont match yaml
-        names.put("audit_logging_options_logger", "audit_logging_options.logger.class_name");
-        names.put("server_encryption_options_client_auth", "server_encryption_options.require_client_auth");
-        names.put("server_encryption_options_endpoint_verification", "server_encryption_options.require_endpoint_verification");
-        names.put("server_encryption_options_legacy_ssl_storage_port", "server_encryption_options.legacy_ssl_storage_port_enabled");
-        names.put("server_encryption_options_protocol", "server_encryption_options.accepted_protocols");
-
-        // matching names
-        names.put("audit_logging_options_audit_logs_dir", "audit_logging_options.audit_logs_dir");
-        names.put("audit_logging_options_enabled", "audit_logging_options.enabled");
-        names.put("audit_logging_options_excluded_categories", "audit_logging_options.excluded_categories");
-        names.put("audit_logging_options_excluded_keyspaces", "audit_logging_options.excluded_keyspaces");
-        names.put("audit_logging_options_excluded_users", "audit_logging_options.excluded_users");
-        names.put("audit_logging_options_included_categories", "audit_logging_options.included_categories");
-        names.put("audit_logging_options_included_keyspaces", "audit_logging_options.included_keyspaces");
-        names.put("audit_logging_options_included_users", "audit_logging_options.included_users");
-        names.put("server_encryption_options_algorithm", "server_encryption_options.algorithm");
-        names.put("server_encryption_options_cipher_suites", "server_encryption_options.cipher_suites");
-        names.put("server_encryption_options_enabled", "server_encryption_options.enabled");
-        names.put("server_encryption_options_internode_encryption", "server_encryption_options.internode_encryption");
-        names.put("server_encryption_options_optional", "server_encryption_options.optional");
-        names.put("transparent_data_encryption_options_chunk_length_kb", "transparent_data_encryption_options.chunk_length_kb");
-        names.put("transparent_data_encryption_options_cipher", "transparent_data_encryption_options.cipher");
-        names.put("transparent_data_encryption_options_enabled", "transparent_data_encryption_options.enabled");
-        names.put("transparent_data_encryption_options_iv_length", "transparent_data_encryption_options.iv_length");
-
-        return names;
```

Comment:
why are you removing?  These strings only make sense to this class, so pulling out to a config registry is not safe, only this class should know about config names it created in the past